src/ci/github-actions/calculate-job-matrix.py

```diff
@@ -136,7 +136,7 @@ def calculate_jobs(run_type: WorkflowRunType, job_data: Dict[str, Any]) -> List[
                 raise Exception(f"Custom job(s) `{unknown_jobs}` not found in auto jobs")
 
         return add_base_env(name_jobs(jobs, "try"), job_data["envs"]["try"])
-    elif run_type is AutoRunType:
+    elif isinstance(run_type, AutoRunType):
```

Comment:
This is the bug that caused the PR to run no jobs.

Reply:
Whoops.